hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnector.java

```diff
@@ -507,7 +507,7 @@ public Vertex sinkProcessor(@Nonnull DagBuildContext context) {
      */
     @SuppressWarnings("ReturnCount")
     private static QueryDataType resolveType(String columnTypeName) {
-        switch (columnTypeName.toUpperCase()) {
+        switch (columnTypeName.toUpperCase(Locale.ROOT)) {
```

Comment:
Valid change, but unrelated to the PR.

Reply:
Can we revert this and send as a separate PR? I am sure there are other cases of `toUpper/LowerCase()` in the codebase.